library/data/registry_entry_attachment/src/testmod/java/org/quiltmc/qsl/registry/attachment/test/AttachmentTestUtil.java

```diff
@@ -0,0 +1,45 @@
+package org.quiltmc.qsl.registry.attachment.test;
+
+import org.quiltmc.qsl.registry.attachment.api.RegistryEntryAttachment;
+import org.quiltmc.qsl.registry.attachment.api.RegistryExtensions;
+
+import net.minecraft.item.Item;
+import net.minecraft.registry.Registries;
+import net.minecraft.registry.Registry;
+import net.minecraft.registry.RegistryKey;
+import net.minecraft.registry.RegistryKeys;
+import net.minecraft.util.Identifier;
+
+import java.util.function.Function;
+
+public final class AttachmentTestUtil {
+	private AttachmentTestUtil() {
+		throw new AssertionError(AttachmentTestUtil.class.getSimpleName() + " contains only static members");
```

Comment:
I think we usually used illegal state exception or something else. I would look at other static classes.